website/templates/base.mako

```diff
@@ -50,6 +50,27 @@
 
 </head>
 <body data-spy="scroll" data-target=".scrollspy">
+
+    % if private_link_anonymous:
+      ## z-index set to 8 to not overlap with getting started with osf non-logged in user bottom-pop-up
+      ## bottom = 30px to not completely block copyright information
+      <style>
```

Comment:
Is it common practice to drop &lt;style&gt; elements mid-stream like this? It's an anti-pattern.